scripts/build/set-project-version.ps1

```diff
@@ -0,0 +1,10 @@
+# Read the project version from the Version.props file
```

Comment:
Let's rename this file as we also have `scripts/version/set-version.ps1`. The names are too close.
Ideas (I don't like any, but it might inspire you :) ):
`set-azp-project-version-variable` - az the `##vso` is bound with AZP
`set-azp-environment-variables` - just to be more generic, we might put more stuff here in the future
`init-azp-environment-variables`
`set-ci-environment-variables`